doc/source/tutorial/stats/continuous_genhyperbolic.rst

```diff
@@ -0,0 +1,139 @@
+
+.. _continuous-genhyperbolic:
+
+Generalized Hyperbolic Distribution
+===================================
+
+The Generalized Hyperbolic Distribution is defined as the normal variance-mean mixture with Generalized Inverse Gaussian distribution as the mixing distribution.
+The "hyperbolic" characterization refers to the fact that the shape of the log-probability distribution can be described as a hyperbola. Hyperbolic distributions are sometime referred to as semi-fat tailed because their probability density decrease slower than "sub-hyperbolic" distributions (e.g. normal distribution, whose log-probability decreases quadratically), but faster than other "extreme value" distributions (e.g. `pareto` distribution, whose log-probability decreases logarithmically).
+
+Functions
+---------
+
+Different parameterization exists in the literature; SciPy implements the location-scale as in Prause (1999).
```

Comment:
```suggestion
Different parameterizations exist in the literature; SciPy implements the "4th parametrization" in Prause (1999).
```
I couldn't find "location-scale" in the Prause reference cited below, but this looks like the "4th parametrization" in that paper.

Note that year 1999 isn't mentioned in the references below, so maybe there was a different paper? But we should use the references we cite and vice-versa

Reply:
You are right [this](https://freidok.uni-freiburg.de/fedora/objects/freidok:15/datastreams/FILE1/content) is the phd dissertation (not a paper, at least not that i could find) from which the 2002 paper is likely taken, I will reference it here and in the docstring.